src/sonic-yang-models/yang-models/sonic-logging.yang

```diff
@@ -0,0 +1,62 @@
+module sonic-logging {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-logging";
+    prefix logging;
+
+    description "LOGGING YANG Module for SONiC-based OS";
+
+    revision 2022-08-27 {
+        description "First Revision";
+    }
+
+    container sonic-logging {
+
+        container LOGGING {
+
+            description "LOGGING part of config_db.json";
+
+            list LOGGING_LIST {
+
+                key "name";
+
+                leaf name {
+                    type string {
+                        length 1..255;
+                    }
+                }
+
+                leaf disk_percentage {
+                    type decimal64 {
+                        fraction-digits 3;
+                        range 0.001..100;
+                    }
+                }
+
+                leaf frequency {
+                    type string {
+                        pattern "daily|weekly|monthly|yearly";
```

Comment:
It is not like that. logrotate tool is really running very often, but it doesn't give any result. It only rotates files in case it will exceed `size` parameter, or age of file is higher than `frequency`. Moreover, we can't add more values here because it came from linux : https://linux.die.net/man/8/logrotate#:~:text=Log%20files%20are%20rotated%20when%20they%20grow%20bigger%20than%20size%20bytes